ocaml/xapi/xapi_cluster_helpers.ml

```diff
@@ -103,3 +103,60 @@ let with_cluster_operation ~__context ~(self : [`Cluster] API.Ref.t) ~doc ~op
           (Datamodel_common._cluster, Ref.string_of self)
       with _ -> ()
   )
+
+let maybe_generate_alert ~__context ~num_hosts ~missing_hosts ~new_hosts ~quorum
+    =
+  List.iter
```

Comment:
I'd like to also check the experimental feature flag here first.

Reply:
ok, same restrict_cluster_health feature flag then?